node/node.go

```diff
@@ -24,8 +25,8 @@ type Node struct {
 	// the Node keeps a reference to the DI App that controls the lifecycles of services registered on the Node.
 	app *fx.App
 
-	// the RPC client is only necessary for full nodes
-	RPCClient *rpc.Client `optional:"true"`
+	// CoreClient provides access to Core Node
```

Comment:
```suggestion
	// CoreClient represents an either embedded or remote Core node's RPC interface.
```

Reply:
That's not true, as when embedded it is not an RPC. RPC by definition is RemotePC

Reply:
That's not correct as well. Your statement says that "CoreClient ___provides___ access ... to node's Client", when CoreClient ___is___ the Client interface itself.

Reply:
```suggestion
	// CoreClient provides access to a Core node process.
```